app/webpacker/controllers/super_selector_controller.js

```diff
@@ -0,0 +1,18 @@
+import ApplicationController from "./application_controller";
+
+export default class extends ApplicationController {
+  connect() {
+    super.connect();
+    window.addEventListener("click", this.handleClick);
+  }
+  disconnect() {
+    super.disconnect();
+    window.removeEventListener("click", this.handleClick);
+  }
+
+  handleClick = (event) => {
```

Comment:
I'm wondering if `handleClick` could be renamed to what it does, like `closeOnClickOutside`.

Reply:
Right, done!